### PR TITLE
[reactstrap] Update type of CustomInput label prop

### PR DESCRIPTION
**Pull request by bettyyin**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/reactstrap/reactstrap/blob/317e8014e30e2084ad02424f69d6f840730b452a/src/CustomInput.js#L10
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/reactstrap/lib/CustomInput.d.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ export type CustomInputType =
 
 export type CustomInputProps<T = {}> = React.InputHTMLAttributes<HTMLInputElement> & {
   type: CustomInputType;
-  label?: string;
+  label?: React.ReactChild;
```
Comment (changLiuUNSW) [inline types/reactstrap/lib/CustomInput.d.ts]: The type is wrong. It should be ```React.ReactNode```

Comment (changLiuUNSW) [reply types/reactstrap/lib/CustomInput.d.ts]: test

Comment (changLiuUNSW) [inline types/reactstrap/lib/CustomInput.d.ts]: The type is wrong. It should be ```React.ReactNode```

**Review by changLiuUNSW — CHANGES_REQUESTED** [review]

**Review by changLiuUNSW — APPROVED** [review]

**Review by uniqueiniquity — CHANGES_REQUESTED** [review]
Please add a test to validate your change.

**Comment by changLiuUNSW** [comment]
@bettyyin  Can you grand me the access for your repo and I can fix the issue for you if you do not have time

**Comment by changLiuUNSW** [comment]
@uniqueiniquity  I will create a new PR if @bettyyin did not have time to fix the issue

**Pull request closed without merging**